waypoint_generator/src/waypoint_generator/waypoint_generator_main.cpp

```diff
@@ -0,0 +1,21 @@
+/*
+* Copyright (C) 2019 LEIDOS.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*/
+
+#include <waypoint_generator/waypoint_generator_node.hpp>
+
+int main(int argc, char** argv)
+{
+}
```

Comment:
I guess you want to add something here?

Reply:
Whoops, yeah, the whole node startup basically. A few lines, but some very important lines...